### PR TITLE
guide on badges

### PR DESCRIPTION
**Pull request by casperdcl**
Been meaning to do this for ages.

(Finally) closes https://github.com/iterative/dvc.org/issues/234.

### CODE REVIEW

File: content/docs/command-reference/metrics/index.md [hunk 1]
```diff
@@ -92,6 +92,16 @@ DVC itself does not ascribe any specific meaning for these numbers. Usually they
 are produced by the model training or model evaluation code and serve as a way
 to compare and pick the best performing experiment.
 
+<admon type="tip">
+
+See
+[the guide on metrics badges](/doc/start/data-management/metrics-parameters-plots#badges)
+for prettifying a repository's with an automatically updated badge like this:
+
+[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
```
Comment (casperdcl) [inline content/docs/command-reference/metrics/index.md:101]:
```suggestion
[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737&logo=DVC)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
```

File: content/docs/start/data-management/metrics-parameters-plots.md [hunk 2]
```diff
@@ -290,5 +290,32 @@ file:///Users/dvc/example-get-started/plots.html
...
+
+So a GitHub markdown example for
+[iterative/example-get-started@main/evaluation.json](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
+would be:
+
+```md
+[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
+```
+
+[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
```
Comment (casperdcl) [inline content/docs/start/data-management/metrics-parameters-plots.md:316]:
````suggestion
[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737&logo=DVC)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
```

[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737&logo=DVC)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
````

**Review by casperdcl — COMMENTED** [review]
potentially with DVC logo?

File: content/docs/command-reference/metrics/index.md (hunk 1 above)
Comment (shcheklein) [reply content/docs/command-reference/metrics/index.md:101]: Yep, if we show a single badge, it makes to include logo. I the guide we can put a link on how to customize it

File: content/docs/start/data-management/metrics-parameters-plots.md [hunk 3]
```diff
@@ -290,5 +290,32 @@ file:///Users/dvc/example-get-started/plots.html
 > [Git revisions](https://git-scm.com/docs/gitrevisions) (commits, tags, branch
 > names) to compare.
 
+## Badges
```
Comment (shcheklein) [inline content/docs/start/data-management/metrics-parameters-plots.md:293]: Tbh, this a bit too much for get-started. Let's put it a separate section / page in the UG's experiments?

Comment (shcheklein) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: and we can put a bit more example with DVC logo, etc

**Review by shcheklein — CHANGES_REQUESTED** [review]
Thanks @casperdcl !!

I think it's a bit too much for get started and I would move it into the UG, besides that - it looks cool!

File: content/docs/command-reference/metrics/index.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:101]: The engine adds an "external link" icon to it which doesn't look great

image

Comment (jorgeorpinel) [reply content/docs/command-reference/metrics/index.md:101]: p.s. I also don't think we need this in the cmd ref as it doesn't have much to do with the `metrics` command usage.

Comment (casperdcl) [reply content/docs/command-reference/metrics/index.md:101]:
```suggestion
<a href="https://github.com/iterative/example-get-started/raw/main/evaluation.json"><img alt="DVC-metrics" src="https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737&logo=DVC"/></a>
```

**Review by shcheklein — APPROVED** [review]

File: content/docs/start/data-management/metrics-parameters-plots.md (hunk 3 above)
Comment (casperdcl) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: Do you mean somewhere under https://dvc.org/doc/user-guide or under https://dvc.org/doc/user-guide/experiment-management? The former seems much more appropriate.

Comment (shcheklein) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: > The former seems much more appropriate as badges are for pipeline metrics, not experiment metrics.

they are more or less the same for me (and from the end user perspective I hope)

> Do you mean somewhere under https://dvc.org/doc/user-guide or under https://dvc.org/doc/user-guide/experiment-management

this https://dvc.org/doc/user-guide/experiment-management probably ... or when we have something like Best Practices ... we can mention it in a few places ... we also have https://dvc.org/doc/install/plugins - not directly related, but might make sense to mention there

Comment (casperdcl) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: > they are more or less the same for me (and from the end user perspective I hope)

Very confused by this. I thought:

1. `dvc {stage,repro}`: [pipeline](https://dvc.org/doc/start/data-management/pipelines) DAG similar to makefiles
2. `dvc exp`: ephemeral [experimentation](https://dvc.org/doc/start/experiments)

are two completely and utterly different things? vis. https://github.com/iterative/dvc.org/issues/3630

**Comment by jorgeorpinel** [comment]
Hi. I see this is approved but the docs changes are still the Cmd Ref and Get Started. Are we moving this info to the UG? (I still think that's a much better place.)

**Comment by casperdcl** [comment]
See the open [pending discussion](https://github.com/iterative/dvc.org/pull/3618#discussion_r893337471) - waiting for feedback.

File: content/docs/start/data-management/metrics-parameters-plots.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: Great info but I also agree on not putting it on Get Started. I also don't think we need this in the cmd ref as it doesn't have much to do with the metrics command usage.

Comment (casperdcl) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: > `dvc exp` is just another way to run and capture results

but it won't show up in `metrics.json`. That requires `exp run && exp apply && git commit`... aka `dvc repro && git commit`, right?

Comment (shcheklein) [reply content/docs/start/data-management/metrics-parameters-plots.md:293]: It requires `dvc exp` and `git commit` - similar to `dvc repro` as far as I know.

it's fine to mention about this here, but the whole section is too much ... we can put the details / instruction to plugins and keep some admonition here, wdyt?

**Comment by shcheklein** [comment]
@casperdcl @jorgeorpinel ping

**Comment by jorgeorpinel** [comment]
I still have the same [opinion](https://github.com/iterative/dvc.org/pull/3618#issuecomment-1152996472). Good content but not for the Get Started.

**Comment by shcheklein** [comment]
@casperdcl let's make a section Install/Badges and move it there.

**Comment by shcheklein** [comment]
@casperdcl wdyt? let's please get it done and merge (the best), or merge

**Comment by shcheklein** [comment]
Closing as stale.

**Pull request closed without merging**